benchmarks/utils/gen_model_config_json.py

```diff
@@ -0,0 +1,76 @@
+import argparse
+import copy
+import json
+import yaml
+
+
+def main():
+
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        "--input",
+        action="store",
+        help="model benchmark config yaml file path",
+    )
+    parser.add_argument(
+        "--output",
+        action="store",
+        help="dir for model benchmark config json file",
+    )
+    arguments = parser.parse_args()
+    convert_yaml_to_json(arguments.input, arguments.output)
+
+def convert_yaml_to_json(yaml_file_path, output_dir):
```

Comment:
So why not just work with json input directly? A new argument needs to be added to many places now